presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/environment/Multinode.java

```diff
@@ -70,7 +74,9 @@ protected void extendEnvironment(Environment.Builder builder)
         builder.configureContainer("presto-master", container -> {
             container
                     .withFileSystemBind(dockerFiles.getDockerFilesHostPath("conf/environment/multinode/multinode-master-jvm.config"), CONTAINER_PRESTO_JVM_CONFIG, READ_ONLY)
-                    .withFileSystemBind(dockerFiles.getDockerFilesHostPath("conf/environment/multinode/multinode-master-config.properties"), CONTAINER_PRESTO_CONFIG_PROPERTIES, READ_ONLY);
+                    .withFileSystemBind(dockerFiles.getDockerFilesHostPath("conf/environment/multinode/multinode-master-config.properties"), CONTAINER_PRESTO_CONFIG_PROPERTIES, READ_ONLY)
+                    .withFileSystemBind(dockerFiles.getDockerFilesHostPath("common/hadoop/hivecached.properties"), CONTAINER_PRESTO_HIVE_CACHED_PROPERTIES, READ_ONLY)
```

Comment:
Please create a new environment, MultinodeHiveCaching

- multinode is not supposed to be "include everything" environment (Iceberg shouldn't be here too)
- in this new environment you can register the catalog as `hive` and run all storage_formats, which greatly improves fs coverage